### PR TITLE
Remove project.json example from versions/index.md

### LINKED ISSUE #2102: Remove project.json example from versions/index.md
There's a *project.json* example lingering in the *versions/index.md* topic + a small textual change to make post v1.0.0 release.

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #2102 

* Replaces the *project.json* example of a patch metapackage with images showing the patch version update for `NETStandard.Library` between `Microsoft.NETCore.App` versions 1.0 and 1.1.
* The phrase "After shipping a .NET Core v1.0.0 stable version" was present prior to the release of v1.0.0 (6/27/2016) and was apparently meant to tell readers at the time that after v1.0.0 comes out the following information applies. Since we're well beyond the release of v1.0.0, I think that phrase can be dropped.

### CODE REVIEW

File: docs/core/versions/index.md [hunk 1]
```diff
@@ -105,24 +105,26 @@ A new version of .NET Core could mean several things:
...
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel" Version="1.1.1" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.1.2" />
+  </ItemGroup>
+</Project>
```
Comment (svick) [inline docs/core/versions/index.md]: This whole page describes versioning of .Net Core, but here you're showing ASP.NET Core, which is technically not part of .Net Core and so I think is not really relevant here.

On the other hand, the two metapackages (`NETStandard.Library` and `Microsoft.NETCore.App`) are not normally visible in a csproj. Maybe remove the example and describe how does msbuild decide which patch version of a metapackage to use?

**Comment by GuardRex** [comment]
@svick Patch versions could possibly be shown in a *.csproj* with `<RuntimeFrameworkVersion>` or `<NetStandardImplicitPackageVersion>`, but how about pics of the deps? This feels good because it shows a metapackage patch update between two minor `Microsoft.NETCore.App` versions.

**Comment by svick** [comment]
@GuardRex Yeah, that makes sense.

**Comment by GuardRex** [comment]
Closing in favor of https://github.com/dotnet/docs/pull/2130

**Pull request closed without merging**